src/collections/_documentation/platforms/javascript/vue.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Vue
+sidebar_order: 40
+---
+
+To use Sentry with your Vue application, you will need to use `@sentry/browser` (Sentry’s browser JavaScript SDK).  
+On its own, `@sentry/browser` will report any uncaught exceptions triggered from your application.
+
+Additionally, the Vue _integration_ will capture the name and props state of the active component where the error was thrown. This is reported via Vue’s `config.errorHandler` hook.
+
+```javascript
+import * as Sentry from '@sentry/browser'
+
+Sentry.init({
+  dsn: '___PUBLIC_DSN___',
+  integrations (integrations) {
+    integrations.push(new Sentry.Integrations.Vue({ Vue }))
```

Comment:
It's worth mentioning that you can skip passing Vue or Ember if they are accessible through window.Vue